dashboard/config/linux/bits/linux-upstream.yml

```diff
@@ -3,4 +3,4 @@
 
 kernel:
  repo: git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
- tag: v5.17-rc1
```

Comment:
Also update linux-next.yml to be no older than the upstream.

Reply:
Ok. <dropping parallel PR>